proto/rill/admin/v1/api.proto

```diff
@@ -1178,6 +1178,7 @@ message UpdateProjectRequest {
   optional string new_name = 9;
   optional int64 prod_ttl_seconds = 10;
   optional string prod_version = 11;
+  optional bool superuser_force_access = 14;
```

Comment:
Doesn't have to be `optional`, the default value when not `optional` will be `false`, which is desired. (`optional` has a different meaning in protos than you would expect – even non-optional fields are not required).